index.html

```diff
@@ -193,14 +193,13 @@
     <h2><dfn>Performance Timeline</dfn></h2>
     <p>Each <a>ECMAScript global environment</a> has:</p>
     <ul>
-      <li>a <dfn>performance observer task queued flag</dfn> and an associated
-      list of <a>registered performance observer</a> objects which is initially
-      empty.
-      </li>
+      <li>a <dfn>performance observer task queued flag</dfn></li>
+      <li>a <dfn>list of <a>registered performance observer</a> objects</dfn> that is initially empty</li>
       <li>a <dfn>performance entry buffer</dfn> to store
-      <a>PerformanceEntry</a> objects. It is initially empty.
+      <a>PerformanceEntry</a> objects that is initially empty
       </li>
     </ul>
+    <p>The <dfn>relevant performance entry buffer</dfn> is the <a>performance entry buffer</a> associated with the <a>relevant global object</a>.</p>
```

Comment:
I find this quite confusing.
There isn't a single buffer, but rather a buffer for each type of entry.
Should the spec reflect this?

Reply:
Re, single buffer: yes, I think we should but that's out of scope for this PR. I flagged this very issue in https://github.com/w3c/performance-timeline/pull/95#issuecomment-336576176 — let's tackle that there. 

In the context of this PR, let's make sure we nail down the correct definition for globals. 

Does `relevant global object` look correct to you?